src/rules/useIsnanRule.ts

```diff
@@ -32,7 +32,8 @@ export class Rule extends Lint.Rules.AbstractRule {
 class UseIsnanRuleWalker extends Lint.RuleWalker {
 
     protected visitBinaryExpression(node: ts.BinaryExpression): void {
-        if (this.isExpressionNaN(node.left) || this.isExpressionNaN(node.right)) {
+        if (this.isExpressionNaN(node.left) || this.isExpressionNaN(node.right)
```

Comment:
So considering operator precedence, this actually evaluates as:

```
this.isExpressionNaN(node.left) || 
(this.isExpressionNaN(node.right) && node.operatoreToken.kind !== ts.SyntaxKind.EqualsToken)
```

The way it's written it kind of implies the opposite. However, I think the behavior is actually correct... you'd never want an `NaN` on the left side, would you?